jest_config/jest.config.json

```diff
@@ -2,25 +2,27 @@
   "rootDir": "../",
   "verbose": true,
   "transform": {
-    "\\.tsx?$": "ts-jest"
+    "^.+\\.jsx?$": "babel-jest",
+    "^.+\\.tsx?$": "ts-jest"
```

Comment:
We can remove `ts-jest` and just use `babel-jest`, e.g.
```
"^.+\\.[t|j]sx?$": "babel-jest"
```
Babel is configured for TS support already.

Reply:
Following your suggestion it has been removed in favor of defaults